src/Controls/src/Build.Tasks/NodeILExtensions.cs

```diff
@@ -156,6 +156,8 @@ static T TryFormat<T>(Func<string, T> func, IXmlLineInfo lineInfo, string str)
 					{ module.ImportReference(("Microsoft.Maui", "Microsoft.Maui.Converters", "CornerRadiusTypeConverter")), typeof(CornerRadiusTypeConverter) },
 					{ module.ImportReference(("Microsoft.Maui", "Microsoft.Maui.Converters", "EasingTypeConverter")), typeof(EasingTypeConverter) },
 					{ module.ImportReference(("Microsoft.Maui.Graphics", "Microsoft.Maui.Graphics.Converters", "ColorTypeConverter")), typeof(ColorTypeConverter) },
+					{ module.ImportReference(("Microsoft.Maui.Graphics", "Microsoft.Maui.Graphics.Converters", "RectTypeConverter")), typeof(RectangleTypeConverter) },
+					{ module.ImportReference(("Microsoft.Maui.Controls", "Microsoft.Maui.Controls", "BrushTypeConverter")), typeof(XamlC.BrushTypeConverter) },
```

Comment:
This line was picking the wrong type, `Microsoft.Maui.Controls.BrushTypeConverter`, so I put `XamlC.`.

Reply:
there's no need to add a known type converter. we keep this for exceptions. the correct way is to attribute the (non-compiled) converter with `[ProvideCompiled]`